jbpm-wb-executor-service/jbpm-wb-executor-service-backend/src/main/java/org/jbpm/workbench/es/backend/server/DataSetDefsBootstrap.java

```diff
@@ -84,6 +85,50 @@ protected void registerDataSetDefinitions() {
         // Register the data set definitions
         dataSetDefRegistry.registerDataSetDef(requestListDef);
         logger.info("Executor service datasets registered");
+
+        DataSetDef executionErrorListDef = DataSetDefFactory.newSQLDataSetDef()
+                .uuid(EXECUTION_ERROR_LIST_DATASET)
+                .name("Error Management")
+                .dataSource(JBPM_DATA_SOURCE)
+                .dbSQL("select "
+                               + "eri.ERROR_ACK, "
+                               + "eri.ERROR_ACK_BY, "
+                               + "eri.ERROR_ACK_AT, "
+                               + "eri.ACTIVITY_ID, "
+                               + "eri.ACTIVITY_NAME, "
+                               + "eri.DEPLOYMENT_ID, "
+                               + "eri.ERROR_DATE, "
+                               + "eri.ERROR_ID, "
+                               + "eri.ERROR_MSG, "
+                               + "eri.JOB_ID, "
+                               + "eri.PROCESS_ID, "
+                               + "eri.PROCESS_INST_ID, "
+                               + "eri.ERROR_TYPE "
+                               + "from "
+                               + "ExecutionErrorInfo eri ",
```

Comment:
This column would contain stacktraces of errors.. long strings, that I don't thins make sense to show at the table. or allow filter by that. For that reason it's only load when the error details is requested with the remote specific call getError.